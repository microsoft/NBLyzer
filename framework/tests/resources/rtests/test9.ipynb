{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "x = 0"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "y = x"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "z = x"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "t = y + z\n",
    "m = x"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "x = t"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3.10.0 64-bit",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "name": "python",
   "version": "3.10.0"
  },
  "orig_nbformat": 4,
  "vscode": {
   "interpreter": {
    "hash": "2469a70536e4d2335a2ea8907942d0699c37342a371ac185bdb5b0aa6f073890"
   }
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
