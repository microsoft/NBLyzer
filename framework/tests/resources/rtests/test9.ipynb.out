[{"cell_id":3,"errors":[{"line":1,"label":"t", "error_type":"ErrorType.CRITICAL", "message":"Variable uses outdated values."}],"path":[0, 1, 2, 3]},{"cell_id":4,"errors":[{"line":1,"label":"x", "error_type":"ErrorType.CRITICAL", "message":"Variable uses outdated values."}],"path":[0, 1, 2, 3, 4]}]